Permanent/Views/Cells/ArchiveTableViewCell/ArchiveScreenTopSectionTitleTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ArchiveScreenTopSectionTableViewCell.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 15.09.2021.
+//  Copyright © 2021 Victory Square Partners. All rights reserved.
+//
+
+import UIKit
+
+class ArchiveScreenSectionTitleTableViewCell: UITableViewCell {
```

Comment:
This looks like a duplicate definition of ArchiveScreenSectionTitleTableViewCell, but with a different filename

Reply:
ArchiveScreenTopSectionTitleTableViewCell.swift was deleted.